src/cmd/flux-wreck

```diff
@@ -115,6 +115,23 @@ local function opt_sig (s)
     return tonumber (posix [s])
 end
 
+prog:SubCommand {
+ name = "cancel",
+ description = "Cancel a pending job",
+ usage = "JOBID",
+ handler = function (self, arg)
+    local id = check_jobid_arg (self, arg[1])
+    local resp, err = f:rpc ("sched.cancel", { jobid = id })
```

Comment:
@grondo: I know this PR is closed, but I thought I should ask a question as I am testing this.

I'm using `flux_request_unpack ()` to unpack this RPC request within flux-sched, and I initially used `{ s:I }` to fetch the value of `id`. But this this led to a protocol error.

Then, I realized what's needed is =`{ s:s }` for conversion.
  
I can always covert the string into `int64_t`, but I thought I would ask if this is intended.

Reply:
Sorry, I thought that check_jobid_arg() converted id to a Lua number and thus the value would be encoded as int64. I'll fix this up for you and sorry I didn't catch it!

Reply:
Not a problem! This is just who will do the conversion.

Reply:
For now you can try changing the code to 
```lua
  f:rpc ("sched.cancel", { jobid = tonumber (id) }
```

And see if that works